cli/config/configuration.toml.in

```diff
@@ -134,6 +134,12 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # used for 9p packet payload.
 #msize_9p = @DEFMSIZE9P@
 
+# If true and vsocks are supported, use vsocks to communicate directly
+# with the agent and no proxy is started, otherwise use unix
+# sockets and start a proxy to communicate with the agent.
+# Default false
+use_vsock = true
```

Comment:
If default is false (which I agree with), we should comment this line, don't you think ?

Reply:
`false` if it's not specified, but I'd prefer to have this enabled in the configuration file, since I consider `vsock` is more stable than `yamux`

Reply:
anyway if `vsock` is not supported, a serial port is used

Reply:
Actually, it would be better `off` by default, because I was thinking that we don't have the agent change yet, and we don't want ppl to have the code trying to use vsock when the agent might not support it yet.

Reply:
If we want it to be enabled by default, please change the comment to say `Default true` instead.

OTOH, I do agree with @sboeuf we should default to false right now. We can change it to true when both runtime and agent changes are merged and we have found out these changes are stable enough.

Reply:
@devimc you haven't addressed this comment.

Reply:
`vsock` works with the current agent code

Reply:
@devimc -- I tend to agree with @sboeuf  here.  Let's get the functionality in, keeping the current behavior (virtio-serial) by default.  We can do a follow on PR to enable by default once we have the CI and performance well understood.  WDYT?